<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>496</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>496</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UnitedChanger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LoginLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login (email):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginEnter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEnter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="ForgotPasswordBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>222</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>222</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't have an account?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>322</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
